src/main/java/http/handlers/auth/AuthHandler.java

```diff
@@ -0,0 +1,38 @@
+package http.handlers.auth;
+
+import http.controllers.IController;
+import http.request.HttpRequest;
+
+import java.util.Base64;
+
+public class AuthHandler implements IAuth {
+
+    private String userName;
+    private String password;
+
+    public AuthHandler() {
+        this.userName = "";
+        this.password = "";
+    }
+
+    @Override
+    public void setAuthCredentials(String userName, String password) {
```

Comment:
why not do this as part of the constructor?

Reply:
👍 makes sense.